api/src/public/login/style.css

```
@@ -147,6 +147,38 @@ form {
   text-decoration: none;
 }
 
+#password-container {
+  position: relative;
+  height: 40px;
+}
+
+#password-container #password {
+  width: 245px;
+  padding: 6px 43px 6px 10px;
+}
+
+#password-container #password-toggle {
+  position: absolute;
+  right: 0px;
+  top: 0px;
```

Nit: Slightly cleaner to not use units for 0 values.

```suggestion
  right: 0;
  top: 0;
```